### PR TITLE
Update README.md

### LINKED ISSUE #148: [Feat]: Improve README
### My suggestion

- Reorganize README
- Add more lore about Nasa Space Apps, how it went for us etc.
- Mention that code quality isn't the best
- Overhaul it

### Possible alternatives

_No response_

### PR DESCRIPTION
**Pull request by wojpo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,54 @@ Project made by EKK Ingarden for [Nasa Space Apps 2024](https://www.spaceappscha
 
 https://github.com/user-attachments/assets/c1c3b7da-9f4d-4ddc-b531-02a951505b03
```
Comment (Norbiros) [inline README.md:5]: Move this to the bottom

File: README.md [hunk 2]
```diff
@@ -4,6 +4,54 @@ Project made by EKK Ingarden for [Nasa Space Apps 2024](https://www.spaceappscha
...
 
+Note: This project was developed as part of a hackathon. While we strived to deliver a functional and engaging application, the code quality may not meet production standards due to time constraints.
```
Comment (Norbiros) [inline README.md:7]: Use:
> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,54 @@ Project made by EKK Ingarden for [Nasa Space Apps 2024](https://www.spaceappscha
...
+
+
+# 📚 Overview
+The Landsat Web Tracker bridges the gap between ground-based spectral measurements and Landsat Surface Reflectance (SR) data. By providing users with an intuitive web-based interface, the application makes satellite data more accessible and actionable, fostering environmental awareness and empowering individuals to contribute toward sustainability.
+Our solution allows users to:
+
+- Select a target location and time range.
```
Comment (Norbiros) [inline README.md:14]: This is kinda duplicate of Features section, merge those sections together

File: README.md [hunk 4]
```diff
@@ -4,6 +4,54 @@ Project made by EKK Ingarden for [Nasa Space Apps 2024](https://www.spaceappscha
...
+- Receive notifications when Landsat satellites pass over their chosen location.
+- Access and visualize Landsat data, including key environmental metrics like temperature, cloud coverage, and Earth-Sun distance.
+- Generate reports with advanced metadata, band views, and reflectance analyses.
+- Download data in PDF and CSV formats.
```
Comment (Norbiros) [inline README.md:18]: That's not entirely true

**Review by Norbiros — CHANGES_REQUESTED** [review]
Also maybe create section about the hackathon, where you add link to task and give more inside how we worked on this project, about results etc.

File: README.md (hunk 3 above)
Comment (wojpo) [reply README.md:14]: Its overview features section expanse it

File: README.md (hunk 4 above)
Comment (Norbiros) [reply README.md:17]: Umm, so let's not spread this lie even further :3

File: README.md (hunk 3 above)
Comment (Norbiros) [reply README.md:13]: Kinda, but I don't see a reason to put shorter and longer versions, just combine them into one. If this section would be a lot shorter okay, but both sections are a bit long.

Comment (wojpo) [reply README.md:13]: huh did u read them?

File: README.md (hunk 4 above)
Comment (wojpo) [reply README.md:17]: ok so tell me which stuff is true and which is not

**Review by Sanielia — DISMISSED** [review]
delete false information, rest lgtm, also overview should have shorter version of features for people who are to lazy, so @Norbiros is over party

**Review by Norbiros — APPROVED** [review]

**Pull request merged**